atomic_reactor/plugins/pre_bump_release.py

```diff
@@ -23,18 +23,24 @@ class BumpReleasePlugin(PreBuildPlugin):
     key = "bump_release"
     is_allowed_to_fail = False  # We really want to stop the process
 
-    def __init__(self, tasker, workflow, target, hub):
+    # The target parameter is no longer used by this plugin. It's
+    # left as an optional parameter to allow a graceful transition
+    # in osbs-client.
+    def __init__(self, tasker, workflow, target=None, hub=None):
```

Comment:
Actually plugin parameters are always passed by keyword, so it's fine to swap target and hub around and only make target optional.